src/php/Api/Infrastructure/PhelFnLoader.php

```diff
@@ -17,13 +18,176 @@
 
 final readonly class PhelFnLoader implements PhelFnLoaderInterface
 {
+    private const PRIVATE_SYMBOLS = [
+        Symbol::NAME_APPLY => [
+            'doc' => '',
+            'fnSignature' => '',
+            'desc' => '',
+        ],
+        Symbol::NAME_CONCAT => [
+            'doc' => '',
+            'fnSignature' => '',
+            'desc' => '',
+        ],
+        Symbol::NAME_DEF => [
+            'doc' => '',
+            'fnSignature' => '',
+            'desc' => '',
+        ],
+        Symbol::NAME_DEF_STRUCT => [
+            'doc' => '',
+            'fnSignature' => '',
+            'desc' => '',
+        ],
+        Symbol::NAME_DO => [
+            'doc' => '',
+            'fnSignature' => '',
+            'desc' => '',
+        ],
+        Symbol::NAME_FN => [
+            'doc' => '',
+            'fnSignature' => '',
+            'desc' => '',
+        ],
+        Symbol::NAME_FOREACH => [
+            'doc' => '',
+            'fnSignature' => '',
+            'desc' => '',
+        ],
+        Symbol::NAME_IF => [
+            'doc' => '```phel
+(if test then else?)
+```
+A control flow structure. First evaluates test. If test evaluates to true, only the then form is evaluated and the result is returned. If test evaluates to false only the else form is evaluated and the result is returned. If no else form is given, nil will be returned.
+
+The test evaluates to false if its value is false or equal to nil. Every other value evaluates to true. In sense of PHP this means (test != null && test !== false).',
+            'fnSignature' => '(if test then else?)',
+            'desc' => 'A control flow structure. First evaluates test. If test evaluates to true, only the then form is evaluated and the result is returned. If test evaluates to false only the else form is evaluated and the result is returned. If no else form is given, nil will be returned.',
```

Comment:
Is there any reason why only `NAME_IF` has the documentation and other fields filled while the others do not?

Reply:
yes, it is WIP, I am working on it ;)